annotations/views.py

```diff
@@ -268,12 +268,14 @@ def list_user(request):
     sort_dict = {"user_name":"username", "name":"full_name",
```

Comment:
The reason is we do not want to expose our table structure to the outside world. It might seem trivial but you never know when it might be used against you. Its just a good practice I read somewhere. I can change it if you feel otherwise. :-)